pkg/cdi/spec_test.go

```diff
@@ -525,7 +525,7 @@ func specType(content []byte) string {
 }
 
 func TestCurrentVersionIsValid(t *testing.T) {
-	require.NoError(t, validateVersion(cdi.CurrentVersion))
+	require.NoError(t, cdi.ValidateVersion(&cdi.Spec{Version: cdi.CurrentVersion}))
 }
 
 func TestRequiredVersion(t *testing.T) {
```

Comment:
We could consider moving these to `specs-go/version_test.go`.

Reply:
This would add a bunch of dependencies to specs-go/go.mod/sum, which would make it harder to import into k/k

Reply:
That is true. Let's leave it where it is then.